### DISCUSSION TITLE
Key Word Spotting | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1423] [Q&A]
# Key Word Spotting | Seeed Studio Wiki

Key Word Spotting

https://wiki.seeedstudio.com/tinyml_course_Key_Word_Spotting/

**Comment by HotRay405** [comment]
I'm having problem with this display. 
I'm using Xiao ESP32S3 sense, and every time I upload a sketch from Arduino IDE, 
it uploads correctly, but when I turn on the display, it doesn't show anything (there's a little bit of backlighting). 
Can anyone help?

**Reply by HotRay405** [reply]
sorry wrong post
